assets/userCtrl.js

```diff
@@ -285,13 +289,20 @@ module.exports = function ($scope, $location, io) {
       var game = $scope.user.flairGames[j];
       if (game.ign) {
         hasIGN = true;
+        if (game.ign.length > 12) {
```

Comment:
We could do this, but we might have to sacrifice some specificity. The client has explicit fields for FCs/IGNs/games, whereas the server has to extract the info from a (possibly-malformed) flair text. So the client is able to say "this game contains an illegal character" whereas the server is sometimes only be able to say, "there's a formatting error but I'm not sure what it is" because the illegal character messes up the parsing.